src/pages/Login.jsx

```
@@ -43,14 +75,71 @@ export default function Login() {
           className="w-full p-2 border rounded"
         />
         <div className="flex items-center justify-between">
-          <button className="px-4 py-2 bg-blue-600 text-white rounded">
+          <button className="inline-flex justify-center py-2 px-6 border border-gray-300 rounded-md shadow-sm bg-blue-600 text-white cursor-pointer">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)